packages/starlight/__tests__/basics/head.test.ts

```diff
@@ -63,4 +63,29 @@ describe('createHead', () => {
       { tag: 'meta', attrs: { name: 'x' }, content: '' },
     ]);
   });
+
+  test('places the default favicon below any user provided icons', () => {
+    const defaultFavicon = {
+      tag: 'link',
+      attrs: {
+        rel: 'shortcut icon',
+        href: '/favicon.svg',
+        type: 'image/svg+xml'
+      }
+    } as const
+
+    const userFavicon = {
+      tag: 'link',
+      attrs: {
+        rel: 'icon',
+        href: '/favicon.ico',
+        sizes: '32x32'
+      }
+    } as const
+
+    expect(createHead([defaultFavicon, userFavicon])).toMatchObject([
```

Comment:
The way the default and custom head configs are handled in `HeadSEO.astro` is actually more like this:

```suggestion
    expect(createHead([defaultFavicon], [userFavicon])).toMatchObject([
```